pgjdbc/src/main/java/org/postgresql/jdbc/PgDatabaseMetaData.java

```diff
@@ -154,33 +154,58 @@ public boolean nullsAreSortedAtEnd() throws SQLException {
   }
 
   /**
-   * What is the name of this database product - we hope that it is PostgreSQL, so we return that
+   * Retrieves the name of this database product. We hope that it is PostgreSQL, so we return that
    * explicitly.
    *
-   * @return the database product name
-   *
-   * @exception SQLException if a database access error occurs
+   * @return "PostgreSQL"
+   * @throws SQLException if a database access error occurs
    */
   public String getDatabaseProductName() throws SQLException {
     return "PostgreSQL";
   }
 
+  /**
+   * Retrieves the version number of this database product.
+   *
+   * @return database version number
+   * @throws SQLException - if a database access error occurs
+   */
   public String getDatabaseProductVersion() throws SQLException {
     return connection.getDBVersionNumber();
   }
 
-  public String getDriverName() throws SQLException {
-    return "PostgreSQL Native Driver";
+  /**
+   * Retrieves the name of this JDBC driver.
+   *
+   * @return "PostgreSQL JDBC Driver"
```

Comment:
@panchenko have you find a project that use getDriverName() to implement logic and cause a potentially higher impact? I don't have found any.